tests/test_hypothesis.py

```diff
@@ -95,6 +102,10 @@ def test_equality(d):
 @given(d=d, arg1=immutable, arg2=immutable, itemlist=lists(tuples(immutable, immutable), **sz))
 def test_consistency(arity, methodname, B, d, arg1, arg2, itemlist):
     ordered = issubclass(B, OrderedBidirectionalMapping)
+    if ordered and WORKAROUND_OD_NAN_BUG:
+        #assume(nan not in d and nan not in d.values())
+        # TODO: For some reason the assume does not work. Resort to:
+        d = {(k, v) for (k, v) in iteritems(d) if not isnan_(k) and not isnan_(v)}
```

Comment:
Hi @DRMacIver, super quick question in case you have a minute: Is there any reason that `assume(nan not in d and nan not in d.values())` would still result in dictionaries with `nan` in them?